app/views/blasts/index.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="page-header">
+<div class="mt-5 mb-3 border-bottom">
```

Comment:
Should we consider adding a custom page-header class that uses sass to include `mt-5 mb-3 border-bottom`?